frontend/src/types/index.ts

```diff
@@ -39,3 +39,42 @@ export interface DateRangePickerProps {
 export interface GooglePlacesInputProps {
   onLocationSelect: (location: MapData) => void;
 }
+export type Trip = {
```

Comment:
already has TripData

Reply:
TripData is the interface used for the page creating trip; Trip is the data type used for upcoming and the schema. Should I change all of those?